docs/UserGuide.md

```diff
@@ -92,17 +101,78 @@ Adds a task to the planner.
 
    Default: No notes.
 
-#### Example usage 
+##### Command Format
+
+`add n/NAME [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
 
-* `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`
-* `add n/Assignment1 t/1100`
-* `a n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`
+##### Example Usage
 
-#### Expected output 
+The following shows the output from ATHENA after entering `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`.
+You should expect to see a message to confirm that the task is added.
 
-For first example usage `add n/Assignment1 t/1100 D/16-09-2020 d/2 r/Today i/high a/Refer to lecture notes`:
 ![add command screenshot](screenshots/athena%20add.jpg)
 
+#### Intermediate Usage
+
+The `add` command accepts 7 parameters.
+
+* `NAME` is the name of the task. It is a compulsory parameter the user has to input.
+
+The following parameters are optional. If they are left empty, ATHENA will use the default value assigned.
+* `TIME` is the time to start doing this task (**HHMM**). For example, 1100.
+
+   Default: Allocated by Athena (only available for non recurring tasks)
```

Comment:
I think we can put a note to explain what are non-recurring tasks? In case the reader gets confused about this.

Something like

| 💡  Non-recurring tasks refer to tasks with the `RECURRENCE` parameter set to one of ... |
|----------------------------------------------------------------------------------------|